src/Microsoft.TemplateEngine.Orchestrator.RunnableProjects/RunnableProjectGenerator.cs

```diff
@@ -363,6 +362,24 @@ internal static object DataTypeSpecifiedConvertLiteral(IEngineEnvironmentSetting
             }
         }
 
+        private static bool TryResolveChoiceValue(string literal, ITemplateParameter param, out string match)
+        {
+            if (literal == null)
+            {
+                match = null;
+                return false;
+            }
+
+            match = param.Choices.Keys.FirstOrDefault(x => string.Equals(x, literal, StringComparison.OrdinalIgnoreCase));
```

Comment:
feels like this should be `param.Choices.ContainsKey(literal) ? literal : null` if `Choices` can be a case insensitive dictionary